nanoFramework.Json/JsonObject.cs

```diff
@@ -8,15 +8,36 @@
 
 namespace nanoFramework.Json
 {
-    internal class JsonObject : JsonToken
+    /// <summary>
+    /// Represents single JSON object that contains multiple JSON properties.
+    /// </summary>
+    public sealed class JsonObject : JsonToken
     {
         private readonly Hashtable _members = new();
 
+        /// <summary>
+        /// Gets the collection of values from <see cref="_members"/>.
+        /// </summary>
         public ICollection Members => _members.Values;
 
+        /// <summary>
+        /// Adds new key value pair to collection.
+        /// </summary>
+        /// <param name="name">JSON property key.</param>
+        /// <param name="value">JSON property value.</param>
         public void Add(string name, JsonToken value)
         {
             _members.Add(name.ToLower(), new JsonProperty(name, value));
         }
+
+        /// <summary>
+        /// Gets the value of property for given key.
+        /// </summary>
+        /// <param name="name">JSON property key.</param>
+        /// <returns>JsonProperty object which contains key and value of object.</returns>
+        public JsonProperty Get(string name)
+        {
+            return (JsonProperty)_members[name.ToLower()];
```

Comment:
The hastable key has it case already "lowered" in `Add` this conversion `ToLower()` seems like unnecessary and just having a negative impact on performance.

Reply:
It's a "Get" method, we need to keep it the same logic as "Add". Imagine situation when you add property with "Test" key and then try to get it by "Test" string. If we won't use ToLower() you will get exception that key not exists.

Maybe we should get rid of "ToLower()" in both methods?

Reply:
I see your point. 
And yes, maybe we could (should?) get rid of this tolower, especially now that we have that option of dealing with mix case on the property names.

Reply:
I would say that we should. As long as all unit tests will pass. 
Also deleting it should improve overall performance.

Reply:
Removed, All tests are passing.